router-tests/integration_test.go

```diff
@@ -365,6 +367,25 @@ func prettifyJSON(t *testing.T, jsonStr string) string {
 	return res.String()
 }
 
+func TestQueryWithLogging(t *testing.T) {
+	t.Parallel()
+	testenv.Run(t, &testenv.Config{LogObservation: testenv.LogObservationConfig{
+		Enabled:  true,
+		LogLevel: zapcore.InfoLevel,
+	}}, func(t *testing.T, xEnv *testenv.Environment) {
+		res := xEnv.MakeGraphQLRequestOK(testenv.GraphQLRequest{
+			Query: `{ employees { id } }`,
+		})
+		require.JSONEq(t, employeesIDData, res.Body)
+		logEntries := xEnv.Observer().All()
+		require.Len(t, logEntries, 12)
+		natsLogs := xEnv.Observer().FilterMessageSnippet("Nats").All()
```

Comment:
Why do we produce nats logs when nats isn't enabled? 🤔

Reply:
They are all logs of `Nats Event Source enabled`, so I'm assuming it's from the config